### DISCUSSION TITLE
Leveraging intranet authentication

### DISCUSSION THREAD

**Discussion by TechFanDan** [latchset/mod_auth_mellon#79] [Q&A]
I want to implement this on my organization's network where browsers are already authenticated on the network. Can this preexisting authentication be leveraged by this module or would it be ADFS that would make this happen?

Essentially, I imagine visiting the protected URL then being redirected to ADFS, which would immediately redirect to protected URL with environment variables populated without the user having provided input.

**Comment by thijskh** [comment]
Yes this works (in principle at least). The ADFS is the one that decides whether the user is already authenticated. This can be because they entered their credentials a first time and the active session is then reused for other subsequent authentications. Or ADFS is configured in such a way to even reuse a domain logon on a workstation of the user.

The details are ADFS configuration so not on topic here. But indeed this module will work in such a scenario.
